### ISSUE TITLE
Is the implementation a common way to inspect loss landscape?

### ISSUE THREAD

**Issue by jiqiujia** [AlexeyGB/batch_norm_helps_optimization#1]
It seems that your implementation to inspect loss landscape is to train multiple models with different learning rate. However, from the original paper, it says, "we compute the gradient of the loss at that step and measure how the loss changes as we move in that direction". It seems that we shall walk along the direction of current step gradient, and see how the loss changes. The implementation makes me a little confused. Is it a common approximation to inspect the loss landscape?
